wsi/overlaybuffer.h

```diff
@@ -63,6 +63,10 @@ class OverlayBuffer {
 
   virtual uint32_t GetTilingMode() const = 0;
 
+  virtual bool GetInterlace() const = 0;
+
+  virtual void SetInterlace(bool isInterlaced) const = 0;
```

Comment:
Why this function has to be const here

Reply:
They should not be constant.   Thanks for the catch.